airflow/api_connexion/openapi/v1.yaml

```diff
@@ -2186,6 +2171,30 @@ paths:
         '404':
           $ref: '#/components/responses/NotFound'
 
+  /datasets/{uri}:
```

Comment:
@RobbeSneyders  -> this is one you might find interesting - it might be something that you might want to correct in parameter validation implemented by Connexion.

We have (and maybe that is not best) two endpoints here:

* /datasets/{uri}:
* /datasets/events

Where `/datasets/events` returns evnets specific for a dataset and `/dataset/{uri}`  allows to query the datasets by their URI. They obviously overlap in a bad way, but I guess due to back-compatibility it won't be easy to avoid that.  This works fine with OpenAPI in general (correct endpoints are executed when we run either) but there was a problem with Connexion 3 that when the `/dataset/{uri}` was specified before `/dataset/events` - parameter validation thought that `/dataset/events' parameters were wrong (because it toook /dataset/events` parameters as the source of truth) - we fixed it by moving the `/dataset/{uri}` after the `/dataset/events` and it works fine now - but you might want to handle it better:  either by handling the case automatically or flagging when more specific datapoint is specified after the more generic one.

Reply:
Cool. Good it's addressed :).